src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -347,6 +351,16 @@
       <annotation name="org.gtk.GDBus.C.UnixFD" value="true"/>
     </method>
 
+    <!-- Available options:
+         "expect-checksum" (type 's')
```

Comment:
(minor bikeshedding) Just wondering, I would have expected `s: checksum` as a mandatory parameter and then sn optional `b: allow-checksum-mismatch` to turn a checksum-mismatch error into a warning. Any rationale for having this safety check opt-in instead of opt-out?

Reply:
No particular rationale. Yeah, I think in this case it'd make sense to make it opt-out given we want to emphasize correctness.